### ISSUE TITLE
Can't find BeEFAutorun

### ISSUE THREAD

**Issue by Frighyz** [byt3bl33d3r/MITMf#328]
In the "Currently Avaliable Plugins list" BeEFAutorun is listed. However i am not able to find it in -h menu or in the plugins folder. The only thing that I can find that references Beef is the beefapi.py file. Is this a mistake or am I simply missing something?

**Comment by byt3bl33d3r** [comment]
Nope, that's my bad. I frankly forgot where I put the damn thing after I was fixing it.
